apis/kueue/v1beta1/workload_types.go

```diff
@@ -296,6 +314,15 @@ type WorkloadList struct {
 	Items           []Workload `json:"items"`
 }
 
+// QueueingPolicyAction specifies if a suspended job should be queued.
+type QueueingPolicyAction string
+
+const (
+	// QueueingPolicyActionNeverQueue means that we don't queue a workload
+	// after it was suspended
+	QueueingPolicyActionNeverQueue QueueingPolicyAction = "Never"
```

Comment:
QueueingPolicyTypeNever

Reply:
Add a QueueingPolicyTypeAlways for consistency? You might get lint errors if it's not used though